frontend/lib/src/index.ts

```diff
@@ -110,4 +110,5 @@ export { Small } from "./components/shared/TextElements"
 export { spacing, fonts } from "./theme/primitives"
 export { mockTheme } from "./mocks/mockTheme"
 export { default as Alert } from "./components/elements/Alert"
-export { default as Text } from "./components/elements/Text"
\ No newline at end of file
+export { default as TextElement } from "./components/elements/Text"
```

Comment:
I think this is a little odd because there is a `Text` proto message but only `StreamlitLibTest.tsx` uses this `TextElement` so renaming it shouldn't be a big deal and nobody should be using `TextElement` anyways?

Reply:
Yeah, I think that's reasonable!